gaia-rs/tests/abci/scenario_2.rs

```diff
@@ -1,18 +1,150 @@
 use std::path::Path;
 
-use gears::tendermint::types::time::timestamp::Timestamp;
+use gears::{
+    tendermint::types::{proto::crypto::PublicKey, time::timestamp::Timestamp},
+    types::uint::Uint256,
+    utils::node::generate_txs,
+};
+use staking::{CommissionRates, CreateValidator, Description};
 
 use crate::setup_mock_node;
 
 #[test]
-/// In this scenario, we test the initialization of the application and submit a balance transfer on block three.
+/// This scenario has a richer genesis file, with more staking fields.
 fn scenario_2() {
```

Comment:
Should this test be moved to staking tests

Reply:
I don't think so. The state hash has been checked against Gaia, we can't check it so easily without the rest of Gaia.

Reply:
Does this test require any functionality from other xmods? If no, then it should be moved to staking xmod

Reply:
We don't have a checked state hash then

Reply:
For `gaia-rs`? Maybe, but what difference if staking xmod work absolutely fine and tested in own crate?